__tests__/app.test.js

```diff
@@ -24,3 +24,27 @@ describe('GET /api/topics', () => {
     });
 });
 
+
+describe('GET /api/articles', () => {
+    test('200: responds with an array of all articles', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                for (const article of articles) {
+                    expect(article).toMatchObject({
```

Comment:
Because .toMatchObject is a flexible matcher - we need to check that article does **_not_** have a key of body as the instructions ask us to exclude it (see note on getEveryArticles)